internal/services/recoveryservices/backup_policy_vm_resource.go

```diff
@@ -853,6 +936,51 @@ func resourceBackupProtectionPolicyVMSchema() map[string]*pluginsdk.Schema {
 			ValidateFunc: validate.RecoveryServicesVaultName,
 		},
 
+		// `tiering_policy` is defined as the map type in Swagger and currently the key only supports `ArchivedRP`. Service team confirmed that they would support other keys in the future.
+		"tiering_policy": {
+			Type:     pluginsdk.TypeList,
+			MaxItems: 1,
+			Optional: true,
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"archived_rp": {
```

Comment:
this isn't in the docs? should it be remoed?

Reply:
I added it to md file.